src/commands/status.command.ts

```diff
@@ -8,13 +13,14 @@ import { Client, CommandInteraction, Status } from 'discord.js';
 
 import { formatDuration, intervalToDuration } from 'date-fns';
 
-import { Constants } from '../utils/constants';
 import { DiscordMessageService } from '../clients/discord/discord.message.service';
 import { JellyfinService } from '../clients/jellyfin/jellyfin.service';
+import { Constants } from '../utils/constants';
 
 @Command({
   name: 'status',
   description: 'Display the current status for troubleshooting',
+  defaultMemberPermissions: 'ViewChannel',
```

Comment:
Is this intended?

Reply:
Yes, IMO this command should be accessible for everyone.